include/zephyr/arch/riscv/common/linker.ld

```diff
@@ -119,13 +119,6 @@ SECTIONS
 #include <snippets-rom-start.ld>
     } GROUP_LINK_IN(ROMABLE_REGION)
 
-    SECTION_PROLOGUE(_VECTOR_SECTION_NAME,,)
```

Comment:
should remove `_VECTOR_SECTION_NAME` definition at L30 since it is no longer being used

Reply:
Sure, thanks!